.github/workflows/typescript_and_e2e_coverage.yml

```diff
@@ -43,9 +43,9 @@ jobs:
       - name: Check that all e2e test files are captured in wdio.conf.js
         run: python -m scripts.check_e2e_tests_are_captured_in_ci
       - name: Run typescript tests
-        run: python -m scripts.typescript_checks
+        run: python -m scripts.run_typescript_tests
       - name: Run typescript tests in strict mode
-        run: python -m scripts.typescript_checks --strict_checks
+        run: python -m scripts.run_typescript_tests --strict_checks
```

Comment:
These don't run some sorts of tests that we have written, instead, they are running "checks" like lint checks, so I'm not sure whether to consider them as running tests. Also, for "lint checks" scripts we have a `pre_commit_linter` script and not `run_pre_commit_linter_tests`.

Considering:
1. We are moving toward Make script which is already going to be a big shift for oppia developers, I don't think we should make an intermediate change/shift right now to script naming to avoid bad development workflow experiences.

2. Typescript doesn't fall under "tests" and we don't follow `run_..._test` for the checks script, I think it's better to leave it as-is for now.


These are my take but I would add other devworkflow team members in case they want these changes:

@U8NWXD Let me know in case you think these PR changes would be helpful (https://github.com/oppia/oppia/issues/19758). (I can handle reviews in case it's needed!)

Reply:
@DubeySandeep, I brought up this matter in our recent (discussion)[https://github.com/oppia/oppia/discussions/19752]. I found myself confused since the documentation only mentioned it in relation to tests, that's why I raised concern. However, I leave it to your discretion on how to proceed.

Reply:
I suggest run_typescript_checks -- we ought to have a "run".

I think we should rename pre_commit_linter to run_lint_checks, too. I think that one is an odd-one-out and it would be good to standardize.

@DubeySandeep wdyt? (Also note that I'm now point of contact for general dev workflow, not @U8NWXD.)

Reply:
I'm fine with `run_typescript_checks`!